packages/clay-management-toolbar/src/validators.js

```diff
@@ -48,8 +49,16 @@ const filterItemsValidator = Config.arrayOf(Config.shapeOf(filterItemShape));
 
 filterItemShape.items = filterItemsValidator;
 
+const filterLabelsValidator = Config.arrayOf(
+	Config.shapeOf({
+		closeable: Config.bool().value(true),
+		label: Config.string(),
```

Comment:
hey @carloslancha maybe we have to add in **ClayLabel** to support HTML in the `label` API so you can customize it as defined below.

<img width="415" alt="screen shot 2018-08-30 at 21 20 45" src="https://user-images.githubusercontent.com/13750819/44886209-72290780-ac9b-11e8-8288-3b234259801f.png">